ow_faults_detection/README.md

```diff
@@ -0,0 +1,6 @@
+The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+Research and Simulation can be found in README.md in the root directory of
+this repository.
+
+ow_faults_detection
+=========
```

Comment:
Needs fleshing out of course, in particular explanation that this is the lander's more lower-level built-in fault detection as opposed to higher-level/autonomy's.

Reply:
will complete once epic B is complete.